openapi-specs/activities-api-docs.json

```diff
@@ -9417,13 +9416,6 @@
               "$ref": "#/components/schemas/ActivityEligibility"
             }
           },
-          "schedules": {
-            "type": "array",
-            "description": "A list of schedules for this activity. These contain the time slots / recurrence settings for instances of this activity.",
-            "items": {
-              "$ref": "#/components/schemas/ActivitySchedule"
-            }
-          },
```

Comment:
Manually removing this before it gets removed from the Activities API meanse that the activities change won't be a breaking change for us, so we won't have to coordinate releases.